### DISCUSSION TITLE
Inline event view extremely long.

### DISCUSSION THREAD

**Discussion by Dramion** [Vaccarini-Lorenzo/iCalObsidianSync#35] [Q&A]
Is this a bug or how it is meant to be? I am zoomed out quite a bit and can only see up to 1100. Thanks for your time.

<img width="1440" alt="Screenshot 2024-01-17 at 11 25 15 PM" src="https://github.com/Vaccarini-Lorenzo/MagicCalendar/assets/43044187/ae47f30c-0f96-4141-97e8-a192cfdf0eda">

**Comment by Vaccarini-Lorenzo** [comment]
Moved to issues.
